app/src/main/java/science/apolline/service/sensor/IOIOService.kt

```diff
@@ -159,7 +160,7 @@ class IOIOService : ioio.lib.util.android.IOIOService(), AnkoLogger {
                     setServiceStatus(false)
                     error("Unable to start IOIOService: " + e.printStackTrace())
                 }
-                Thread.sleep(freq.toLong())
+                Thread.sleep((COLLECT_DATA_FREQ * 1000).toLong())
```

Comment:
[Magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) should be avoided